lib/read.js

```diff
@@ -95,6 +93,20 @@ class ReadSql extends AdapterRead {
         this.timeField = options.timeField ||
             (options.from || options.to || options.last ? 'time' : undefined);
     }
+    
+    addSorting() {
+        if (this.sortFields) {
+            // adds multiple sort orders
+            this.sortFields.forEach((sortObj) => {
```

Comment:
You're the boss! In my head even though they both use the `order by` clause they're pretty unrelated, I'd rather have all the code that deals with `sort` optimization together.